webapp/channels/src/components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -212,62 +195,35 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
-                            values={{
-                                trialLength: FREEMIUM_TO_ENTERPRISE_TRIAL_LENGTH_DAYS,
-                                highlight: (msg: React.ReactNode) => (
-                                    <strong>{msg}</strong>
-                                ),
-                                linkEvaluation: (msg: React.ReactNode) => (
-                                    <ExternalLink
-                                        location='feature_discovery'
-                                        href={LicenseLinks.SOFTWARE_SERVICES_LICENSE_AGREEMENT}
-                                    >
-                                        {msg}
-                                    </ExternalLink>
-                                ),
-                                linkPrivacy: (msg: React.ReactNode) => (
-                                    <ExternalLink
-                                        location='feature_discovery'
-                                        href={AboutLinks.PRIVACY_POLICY}
-                                    >
-                                        {msg}
-                                    </ExternalLink>
-                                ),
-                            }}
-                        />
-                    ) : (
-                        <FormattedMessage
-                            id='admin.feature_discovery.trial-request.accept-terms'
-                            defaultMessage='By clicking <highlight>Start trial</highlight>, I agree to the <linkEvaluation>Mattermost Software and Services License Agreement</linkEvaluation>, <linkPrivacy>Privacy Policy</linkPrivacy> and receiving product emails.'
-                            values={{
-                                highlight: (msg: React.ReactNode) => (
-                                    <strong>{msg}</strong>
-                                ),
-                                linkEvaluation: (msg: React.ReactNode) => (
-                                    <ExternalLink
-                                        location='feature_discovery'
-                                        href={LicenseLinks.SOFTWARE_SERVICES_LICENSE_AGREEMENT}
-                                    >
-                                        {msg}
-                                    </ExternalLink>
-                                ),
-                                linkPrivacy: (msg: React.ReactNode) => (
-                                    <ExternalLink
-                                        location='feature_discovery'
-                                        href={AboutLinks.PRIVACY_POLICY}
-                                    >
-                                        {msg}
-                                    </ExternalLink>
-                                ),
-                            }}
-                        />
-                    )}
-                </p>}
+                {(!this.props.isCloud) && (<p className='trial-legal-terms'>
```

Comment:
Not sure if this `gettingTrialError` is referring to getting a cloud trial or self-hosted trial. If it's cloud we can probably get rid of that code too below